test/framework/mappers/cr_to_cli.go

```diff
@@ -55,10 +55,8 @@ func GetServiceCLIFlags(serviceHolder *bddtypes.KogitoServiceHolder) []string {
 		cmd = append(cmd, "--svc-labels", fmt.Sprintf("%s=%s", labelName, labelValue))
 	}
 
-	if kogitoRuntime, ok := serviceHolder.KogitoService.(*v1beta1.KogitoRuntime); ok {
-		if runtime := kogitoRuntime.Spec.Runtime; len(runtime) > 0 {
-			cmd = append(cmd, "--runtime", string(runtime))
-		}
+	if runtime := serviceHolder.KogitoService.GetSpec().GetRuntime(); len(runtime) > 0 {
+		cmd = append(cmd, "--runtime", string(runtime))
```

Comment:
```suggestion
	if kogitoRuntime, ok := serviceHolder.KogitoService.(*v1beta1.KogitoRuntime); ok {
		if runtime := kogitoRuntime.GetSpec().GetRuntime(); len(runtime) > 0 {
			cmd = append(cmd, "--runtime", string(runtime))
		}
```

Reply:
`--runtime` flag should be used just for KogitoRuntime, when supporting services like Jobs service are deployed then this flag is not applicable